### ISSUE TITLE
suggestion to clarify logged metrics

### ISSUE THREAD

**Issue by andreaskopp** [swatig007/automlForImages#31]
for image classification, metrics are logged as log_loss_train and accuracy_train.
This is misleading assuming that metrics are actually based on the validation set.

Suggestion:
log training loss / accuracy PLUS validation loss / accuracy for clarity and people can detect potential overfitting

**Comment by raduk** [comment]
We log "accuracy/log_loss" which are computed on the validation set and "accuracy_train/log_loss_train" computed on the train dataset (please see attached image)

Let me know if that answers your questions.

![image](https://user-images.githubusercontent.com/3809388/128785933-d2a10ddd-afc8-4c2e-aa5b-94f06326c652.png)

**Comment by andreaskopp** [comment]
Thanks @raduk. My recommendation to the dev team is to rename accuracy to accuracy_val (same for log_loss) to avoid confusion. 
Of course, people can figure that out in your example above. However, the default experiment view only logs "accuracy". It took me some time to find out that it was actually validation accuracy.
